applications/luci-app-zerotier/root/etc/zerotier.start

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+zero_enable="$(uci get zerotier.sample_config.enabled)"
+
+[ "${zero_enable}" -eq "1" ] || exit 1
+
+count=0
+[ -f "/tmp/zero.log" ] && {
+	while [ -z "$(ifconfig | grep 'zt' | awk '{print $1}')" ]
+	do
+		sleep 2
+		let count++
+		[ "${count}" -lt 5 ] || exit 19
+	done
+}
+
+nat_enable="$(uci get zerotier.sample_config.nat)"
+zt0="$(ifconfig | grep 'zt' | awk '{print $1}')"
+echo "${zt0}" > "/tmp/zt.nif"
```

Comment:
Is this file ever used?

This file just seems a bit messy. [This one](https://github.com/zhengmz/luci-app-zerotier/blob/master/root/etc/init.d/luci-zerotier) looks far more suitable. Consider using it.

Although, those init.d files belong in the zerotier package.

Reply:
yeah this is probably what needs to be done.
thanks for the link